mlrun/feature_store/feature_set.py

```diff
@@ -207,6 +208,13 @@ def targets(self, targets: List[DataTarget]):
     def update_target(self, target: DataTarget):
         self._targets.update(target)
 
+    def update_last_written_for_target(
+        self, target_path: str, last_written: datetime.datetime
+    ):
+        for target in self._targets:
+            if target.path == target_path:
+                target.last_written = last_written
+
```

Comment:
need to update the status? or sync to db somehow?

Reply:
sync to db is done inside ingest